src/node_path.cc

```diff
@@ -0,0 +1,120 @@
+#include "node_path.h"
+#include "env-inl.h"
+#include "node_errors.h"
+#include "node_external_reference.h"
+
+namespace node {
+
+namespace path {
+using v8::Context;
+using v8::FunctionCallbackInfo;
+using v8::Local;
+using v8::Object;
+using v8::Value;
+
+// extracted from
```

Comment:
This probably needs a copy of the license?

Reply:
do you mean just a copy of MIT?

Reply:
No, take a look at `tools/license-builder.sh`